web/models.py

```
@@ -213,6 +214,27 @@
+        if not self.facts:
+            self.facts = []
+        for fact in self.facts:
+            if fact.get("text") == fact_text:
+                return False
+        from django.utils import timezone
+
+        self.facts.append({"text": fact_text, "generated_at": timezone.now().isoformat()})
+        self.save(update_fields=["facts"])
+        return True
+
+    def get_random_fact(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_random_fact`

(ANN201)